src/group_theory/perm/support.lean

```diff
@@ -276,6 +276,40 @@ begin
   { rw [gpow_neg_succ_of_nat, ← support_inv, ← inv_pow, pow_apply_mem_support] }
 end
 
+lemma set_support_inv_eq {α : Type*} (p : perm α) :
```

Comment:
These additions are kind of weird - in the middle of lemmas on the finset `p.support`, where `p` is a permutation of a `fintype`, we get lemmas that hold on a different `set`. Since these results are more general, let's not convert them to the `finset`, but I do want them to live in a separate section of this file. (And you could also instantiate some of the recurring params in a `variables` statement.)